notification/tests.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from django.utils import timezone
+from player.models import Player, BallGame
+from notification.models import Notification, NotificationType
+
+
+@pytest.fixture
+def player():
+    return Player.create(
+        username='Amit',
+        password='password',
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+
+
+@pytest.fixture
+def notification(player):
+    return Notification.objects.create(
+        player_id=player,
+        seen_time=None,
+        sent_time=timezone.now(),
+        message='Test notification',
+        notification_type=NotificationType.WEBSITE,
+        is_read=False
+    )
+
+
+@pytest.mark.django_db
+class TestNotification:
+    def test_create_notification(self, notification):
+        assert isinstance(notification, Notification)
+        assert notification.player_id is not None
+        assert notification.sent_time is not None
+        assert notification.message == 'Test notification'
+        assert notification.notification_type == NotificationType.WEBSITE
+        assert notification.is_read is False
+
+    def test_persisted_notification_in_database(self, notification):
+        assert notification in Notification.objects.all()
```

Comment:
Can you explain me the purpose of this test? Do you want to check if the database acts normally?

Reply:
Yep.
The purpose of this test is to verify if a notification object is correctly persisted in a database.